src/lbaf/IO/lbsVTDataReader.py

```diff
@@ -201,21 +202,21 @@ def _populate_rank(self, phase_id: int, rank_id: int) -> tuple:
                     # Check whether both are objects
                     if c_to.get("type") == "object" and c_from.get("type") == "object":
                         # Create receiver if it does not exist
-                        receiver_obj_id = c_to.get("id")
+                        receiver_obj_id = c_to.get("seq_id")
```

Comment:
We want to be able to read in `"id"` as well, if it's present (i.e. from a file generated by vt, with the bit-encoded ids).

Reply:
It should be ok now